src/NosCore.GameObject/Providers/MinilandProvider/MinilandProvider.cs

```diff
@@ -155,6 +155,8 @@ public void SetState(long characterId, MinilandState state)
                 var ml = _minilandIds[characterId];
                 var miniland = _mapInstanceProvider.GetMapInstance(ml.MapInstanceId);
                 ml.State = state;
+
+                // TODO : If MinilandState.Private => Don't kick friends
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4513002)